src/DistributedComputations/distributed_architectures.jl

```diff
@@ -264,6 +264,11 @@ function Distributed(child_architecture = CPU();
     ranks = Rx, Ry, Rz = size(partition)
     partition_ranks = Rx * Ry * Rz
 
+    if partition_ranks == 1
+        @warn "The script has been launched using a single MPI rank. \n" *
+              "This might not be intended but a result of an incorrect MPI configuration."
```

Comment:
```suggestion
        @warn "We are building a Distributed architecture on a single MPI rank. \n" *
              "This can occur when MPI is incorrectly configured."
```

Reply:
We don't know this comes from a script. So I changed the comment to be more specific.

Is the configuration issue with MPI.jl or MPI itself?

Reply:
Sure, we can point at MPI.jl config docs. It's a problem only of the julia MPI not finding the MPI implementation and acting as if we launch with one rank. So each rank has an independent execution with `MPI.Comm_size == 0` and `MPI.Comm_rank == 0`

Reply:
ok, so can we refine the warning to indicate that MPI.jl is not configured correctly? I think it is confusing to say "MPI is not configured correctly", but hte problem is actually just MPI.jl